modules/support-collecting-network-trace.adoc

```diff
@@ -0,0 +1,114 @@
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+* You have installed the OpenShift CLI (`oc`).
+* You have a Red Hat standard or premium subscription.
+* You have a Red Hat Customer Portal account.
+* You have an existing Red Hat Support case ID.
+
+.Procedure
+
+. Obtain a list of cluster nodes:
++
+----
+$ oc get nodes
+----
+
+. Enter into a debug session on the target node. This step instantiates a debug Pod called `<node_name>-debug`:
++
+----
+$ oc debug node/my-cluster-node
+----
+
+. Set `/host` as the root directory within the debug shell. The debug Pod mounts the host's root file system in `/host` within the Pod. By changing the root directory to `/host`, you can run binaries contained in the host's executable paths:
++
+----
+# chroot /host
+----
++
+[NOTE]
+====
+{product-title} {product-version} cluster nodes running {op-system-first} are immutable and rely on Operators to apply cluster changes. Accessing cluster nodes using SSH is not recommended and nodes will be tainted as _accessed_. However, if the {product-title} API is not available, or the kubelet is not properly functioning on the target node, `oc` operations will be impacted. In such situations, it is possible to access nodes using `ssh core@<node>.<cluster_name>.<base_domain>` instead.
+====
++
+. From within the `chroot` environment console, obtain the node's interface names:
++
+----
+# ip ad
+----
+
+. Start a `toolbox` container, which includes the required binaries and plug-ins to run `sosreport`:
++
+----
+# toolbox
+----
++
+[NOTE]
+====
+If an existing `toolbox` Pod is already running, the `toolbox` command outputs `'toolbox-' already exists. Trying to start...`. To avoid `tcpdump` issues, remove the running toolbox container with `podman rm toolbox-` and spawn a new toolbox container.
+====
++
+. Initiate a `tcpdump` session on the cluster node and redirect output to a capture file. This example uses 'ens5' as the interface name:
++
+----
+tcpdump -nn -s 0 -i ens5 -w /host/var/tmp/my-cluster-node_$(date +%d_%m_%Y-%H_%M_%S-%Z).pcap  <1>
+----
+<1> The `tcpdump` capture file's path is outside of the `chroot` environment because the toolbox container mounts the host’s root directory at `/host`.
+
+. If a `tcpdump` capture is required for a specific container on the node, follow these steps.
+.. Determine the target container ID. The `chroot host` command precedes the `crictl` command in this step because the toolbox container mounts the host's root directory at `/host`:
++
+----
+# chroot /host crictl ps
+----
++
+.. Determine the container's process ID. In this example, the container ID is 'a7fe32346b120':
++
+----
+# chroot /host crictl inspect a7fe32346b120 --output yaml | grep 'pid:' | awk '{print $2}'
```

Comment:
`grep 'pid:'` cannot search the pid.   from the output see
```
chroot /host crictl inspect 62684b5e30802 --output yaml | grep 'pid'
    "pid": 10785
```
should be `pid":`

Reply:
@zhaozhanqi thank you for highlighting this. The output I see in my test does not have double quotes:

~~~
[root@master-0 /]# chroot /host crictl inspect d945dda3a7b94 --output yaml | grep 'pid'
pid: 1642
~~~

Given that what we are seeing is the possibility of two different output formats (`"pid":` and `pid:`, I have changed the line to the following, to accommodate both scenarios:

~~~
# chroot /host crictl inspect a7fe32346b120 --output yaml | grep 'pid' | awk '{print $2}'
~~~

Reply:
oh, it's weird, in my side.  `--output yaml` should be typed before the id (a7fe32346b120), see the difference as below
```
chroot /host crictl inspect -o yaml a69d4f66ad4fe | grep pid
  pid: 3906
      - type: pid
        pids:
```
```
chroot /host crictl inspect  a69d4f66ad4fe -o yaml | grep pid
FATA[0000] Getting the status of the container "-o" failed: rpc error: code = NotFound desc = could not find container "-o": container with ID starting with -o not found: ID does not exist 
    "pid": 3906,
          "pids": {
            "type": "pid"
```

Reply:
@zhaozhanqi that is interesting! I tested each command order and both work for me. The order that works for you follows the `crictl inspect --help` output:

~~~
[root@master-0 /]# chroot /host crictl inspect --help
NAME:
   crictl inspect - Display the status of one or more containers

USAGE:
   crictl inspect [command options] CONTAINER-ID [CONTAINER-ID...]
...
~~~

I have updated the step in the procedure as follows, given that having the option before the container ID works for both of us, and that it is the recommended syntax in the help menu:

~~~
# chroot /host crictl inspect --output yaml a7fe32346b120 | grep 'pid' | awk '{print $2}'
~~~

Please let me know if this is suffice. Thanks!

Reply:
yes, that's LGTM, thanks
/lgtm